### DISCUSSION TITLE
Adding mx margin on Dialog at sm breakpoint

### DISCUSSION THREAD

**Discussion by clintmsand** [shadcn-ui/ui#2640] [Q&A]
Trying to add some simple margin to left and right of a dialog box when viewing on a mobile screen. No matter where I seem to add mx-5 it doesn't actually work. Any ideas on how to add some margin to a shadcn dialog when viewing on a phone like this? <img width="499" alt="Screenshot 2024-01-31 at 8 38 44 PM" src="https://github.com/shadcn-ui/ui/assets/1192447/a1af6bb0-7a72-4e8d-aeb4-caf6c5f4a35d">

**Comment by clintmsand** [comment]
I figured out if you add className="w-11/12 sm:max-w-md" to DialogContent it has the desired effect I was looking for. Closing.

**Reply by ianujjagtap** [reply]
Hey i am also facing the same issue can you please specify where we have to changes 
in the primitive component of the dialouge ?
